onmt/translate/Translator.lua

```diff
@@ -18,7 +18,10 @@ local options = {
   {'-pre_filter_factor', 1, [[Optional, set this only if filter is being used. Before
                             applying filters, hypotheses with top `beamSize * preFilterFactor`
                             scores will be considered. If the returned hypotheses voilate filters,
-                            then set this to a larger value to consider more.]]}
+                            then set this to a larger value to consider more.]]},
+  {'-length_norm', 0.0, [[Length normalization coefficient. If set to 0, no length normalization.]]},
```

Comment:
Could you just be more explicit in the description of these new options? What normalization does and what the coefficient represents? What is EOS penalty?